LEAF_Nexus/sources/System.php

```diff
@@ -242,4 +256,90 @@ public function refreshOrgchartEmployees()
 
         return $ret;
     }
+
+    /**
+     * Get primary admin.
+     *
+     * @return array|false array with primary admin's info
+     */
+    public function getPrimaryAdmin()
+    {
+        $strSQL = "SELECT `data` FROM `settings` WHERE `setting` = 'primaryAdmin'";
+        $primaryAdminRes = $this->db->query($strSQL);
+
+        include_once 'Employee.php';
+        $employee = new Orgchart\Employee($this->db, $this->login);
+
+        if(count($primaryAdminRes))
+        {
+            $user = $employee->lookupLogin($primaryAdminRes[0]['data']);
+            $user = $employee->lookupEmpUID($user[0]['empUID']);
```

Comment:
Am I missing something?  You're assigning $user and then reassigning it.  Looks like maybe you meant $user[]?

Reply:
I just implemented the user search changes to include email so we don't have to do this double method calling anymore.